reps_event_app/lib/ui/repsDetails.dart

```diff
@@ -0,0 +1,83 @@
+import "package:flutter/material.dart";
+import 'package:reps_event_app/models/reps_model.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:latlong/latlong.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class RepsDetails extends StatefulWidget {
+  final RepsModel reps;
+
+  RepsDetails({this.reps});
+
+  @override
+  _RepsDetailsState createState() => _RepsDetailsState();
+}
+
+class _RepsDetailsState extends State<RepsDetails> {
+  bodyWidget(BuildContext context) => Container(
+    child: ListView(
+      scrollDirection: Axis.vertical,
+      children: <Widget>[
+        Container(
+          height: MediaQuery.of(context).size.height/2.75,
+        ),
+        Container(
+            child: Padding(
+                padding: EdgeInsets.symmetric(horizontal: 8, vertical: 8),
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: <Widget>[
+                    Text(
+                      widget.reps.fullname,
+                      textAlign: TextAlign.justify,
+                      style: TextStyle(fontSize: 18),
+                    ),
+                    SizedBox(height: 10,),
+                    Text.rich(TextSpan(children: <TextSpan>[
+                      TextSpan(
+                        text: 'City',
+                        style: TextStyle(
+                            fontStyle: FontStyle.italic,
+                            background: Paint()
+                              ..color = Colors.red[50],
+                            fontSize: 18),
+                      ),
+                      TextSpan(
+                          text: "" +
```

Comment:
What is this empty string `""` for?

Reply:
No idea

Reply:
Oh wait ya, that the field name, example: "city": Vadodara , "": widget.reps.fullname